site/pages/xprmntl/checkboxes-en.hbs

```diff
@@ -0,0 +1,338 @@
+---
+{
+	"altLangPrefix": "checkboxes",
+	"breadcrumb": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html" }
+	],
+	"dateModified": "2020-07-31",
+	"description": "Working example - provisional feature",
+	"language": "en",
+	"parentdir": "checkboxes",
+	"tag": "chckbxrdio",
+	"title": "Checkboxes and radio buttons - Provisional feature"
+}
+---
+{{>alert-provisional-demos}}
+<span class="wb-prettify all-pre hide"></span>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Provisional</dd>
+	<dt>Type</dt>
+	<dd>Canada.ca design pattern</dd>
+	<dt>Last review</dt>
+	<dd>2020-07-31</dd>
+	<dt>Conforming to</dt>
+	<dd>Content and IA spec 2.1</dd>
+	<dt>Specification</dt>
+	<dd><a href="https://design.canada.ca/common-design-patterns/checkboxes-radio-buttons.html">Context-specific features - Canada.ca design pattern</a></dd>
+	<dt>Technical name</dt>
+	<dd>
+		<ul>
+			<li><code>gc-chckbxrdio</code></li>
+		</ul>
+	</dd>
+</dl>
+<h2>Working example</h2>
+<h3>Checkboxes and radio buttons - v.8.0</h3>
+<section>
+	<h4>Large checkboxes target with explicit labels</h4>
+	<fieldset class="provisional gc-chckbxrdio chckbxrdio-grp">
+		<legend>Choose your condiments:</legend>
+		<ul class="list-unstyled lst-spcd-2">
+			<li class="checkbox">
+				<input type="checkbox" id="cond1">
+				<label for="cond1">Ketchup</label>
+			</li>
+			<li class="checkbox">
+				<input type="checkbox" id="cond2">
+				<label for="cond2">Relish</label>
+			</li>
+			<li class="checkbox">
+				<input type="checkbox" id="cond3">
+				<label for="cond3">Mustard</label>
+			</li>
+		</ul>
+	</fieldset>
+	<h4>Large disabled checkboxes target with explicit labels</h4>
+	<fieldset class="provisional gc-chckbxrdio chckbxrdio-grp">
+		<legend>Choose your condiments:</legend>
+		<ul class="list-unstyled lst-spcd-2">
+			<li class="checkbox">
+				<input type="checkbox" id="cond4" disabled>
+				<label for="cond4">Ketchup</label>
+			</li>
+			<li class="checkbox">
+				<input type="checkbox" id="cond5" class="disabled">
+				<label for="cond5">Relish</label>
+			</li>
+			<li class="checkbox">
+				<input type="checkbox" id="cond6" class="disabled" disabled>
+				<label for="cond6">Mustard</label>
+			</li>
+		</ul>
+	</fieldset>
+</section>
+<section>
+	<h4>Single smaller checkbox target with explicit label in an inline form</h4>
+	<form class="form-inline" method="get" action="#">
+		<div class="form-group">
+			<label class="wb-inv" for="exampleInputEmail2">Email address</label>
+			<input type="email" class="form-control" id="exampleInputEmail2" placeholder="Enter email" />
+		</div>
+		<div class="form-group">
+			<label class="wb-inv" for="exampleInputPassword2">Password</label>
+			<input type="password" class="form-control" id="exampleInputPassword2" placeholder="Password" />
+		</div>
+		<div class="checkbox provisional gc-chckbxrdio">
+			<input id="remember2" type="checkbox" class="checkbox-small" />
```

Comment:
In this pattern the user are kind of forced to use the CSS class checkbox-small otherwise there is no spacing between the box and the text.
Is this pattern are only when the CSS class "checkbox" and "gc-chckbxrdio" are used together?

Reply:
Yes, this pattern are only when the CSS class "checkbox-small" and "gc-chckbxrdio" are used together